target_chains/ethereum/contracts/forge-test/Entropy.t.sol

```diff
@@ -1864,8 +1863,7 @@ contract EntropyTest is Test, EntropyTestUtils, EntropyEvents, EntropyEventsV2 {
             assertEq(callbackFailed, true);
             assertEq(callbackErrorCode, bytes(""));
 
-            // callback gas usage is approximate and only triggered when the provider has set a gas limit.
-            // Note: this condition is somewhat janky, but we hit the stack limit so can't put in any more local variables :(
+            // callback gas usage is approximate
```

Comment:
Note that other tests of the `Revealed` event don't check the gas field because it's too hard to predict up front. Thus, we're using `vm.expectEmit` to only check the topics not the data.